assets/js/components/data-table.js

```diff
@@ -71,14 +70,14 @@ export const getDataTableFromData = ( data, headers, options ) => {
 								{ cell }
 							</Link>
 
-							{ showURLs && '' !== trim( link, '/' ) &&
```

Comment:
The check here itself shouldn't be removed, only the `trim` part of it. This should probably become `showURLs && link`.

Reply:
@felixarntz I removed it as I felt it was covered by the ternary operator wrapper from line 62

Reply:
@ShahAaron Oh you're right - sorry about that, all good then.